src/main/scala/chisel3/compatibility.scala

```diff
@@ -438,12 +523,9 @@ package object Chisel {
     }
   }
 
-  @deprecated("debug doesn't do anything in Chisel3 as no pruning happens in the frontend", "chisel3")
-  object debug {
-    def apply(arg: Data): Data = arg
-  }
```

Comment:
Same comment about the below, yes it's been deprecated for years but since we're going to nuke this whole file later, is there any benefit removing it early?

Reply:
mostly i wasn't sure if i could put two @deprecated decorators on the same function, and i wanted people who had already filtered out the earlier warning (for whatever reason) to still see the newer one... trying with 2 deprecation labels